### ISSUE TITLE
Create mosaic not work

### ISSUE THREAD

**Issue by All4Gis** [All4Gis/QGISFMV#11]
TypeError: isinstance expected 2 arguments, got 1

**line 612 QgsFmvUtils**  _dst_ds.SetGeoTransform(geotransform)_

Using the new geotransform matrix not work GeoreferenceFrame method

I think we should use geotransform.to_affine()

**Comment by ltbam** [comment]
I fixed this, but curiously I was not able to find any good transformation with geotransform.to_affine().

homography.to_affine() gives:
8.1737747192, 0.0936541762, -0.0000267846
47.2581863403, 0.55170322997 -0.000008200

and GCP Gives:
8.189739694627354, -0.000019702725, -0.00012723764579571182,
47.230543325282255, -0.00000967247, 0.00011924104111578986)

That's very different :/

I've used GDAL GCP parameters then.

**Keep in mind GCP can only be used for plane transformation, therefore the UAV View axis must be very close to nadir (vertical). Otherwise the georeferenced frame will be wrong.**
